documentation/Query-Syntax.md

```diff
@@ -347,7 +358,7 @@ This expression queries our `/login` endpoint to check a user's credentials, and
 
 ### Authentication
 
-The `GET`, `POST`, and `GraphQL` operators may require authentication for the endpoints they are querying, such as our own [API](API.md). This can be achieved in a couple of different ways.
+The `GET`, `POST`, and `GraphQL` operators may require authentication for the endpoints they are querying, such as our own [API](API). This can be achieved in a couple of different ways.
```

Comment:
Maybe wrong?

Reply:
Yeah, I noticed this too -- the auto-formatter has been changing a lot of the way it formats links lately. I checked some of the existing links on Github wiki that have been changed to this format, and they seem fine, so I think it's okay. Will definitely keep an eye on once we publish the next wiki update!